diff --git src/marin/run/ray_deps.py lib/marin/src/marin/run/ray_deps.py
index 3f061dd73..27798fb44 100644
--- src/marin/run/ray_deps.py
+++ lib/marin/src/marin/run/ray_deps.py
@@ -84,6 +84,8 @@ def compute_frozen_packages(extra: list[str] | None = None) -> PackageSpec:
     cmd = [
         "uv",
         "export",
+        "--package",
+        "marin",
         "--no-default-groups",
         "--no-annotate",
         "--no-hashes",
